### DISCUSSION TITLE
How to display exemplars

### DISCUSSION THREAD

**Discussion by aryagi** [prometheus/client_python#813] [Q&A]
I am trying the three step demo in the readme and added the example code for a exemplar in a counter metrics in the exemplar section. When I viewed the metrics page, i am able to see the created counter metric and its value. However, the exemplar is not showing up. I would like to know if there is anything additional necessary to display exemplars.

**Comment by roidelapluie** [comment]
You need the HTTP header `Accept: application/openmetrics-text`
